### DISCUSSION TITLE
ANDROID L3 DEVICES CAN PLAY VIDEOS FROM MAIN PROFILE ONLY

### DISCUSSION THREAD

**Discussion by George-R-Young** [CastagnaIT/plugin.video.netflix#1457] [General]
This is true for your plugin and my L3 Android box.

However, my old (Version 6.26.1 build 1531696) Android Netflix app on the same box can play videos from any profile.

It looks like a plugin limitation rather than a Netflix server limitation.

**Comment by CastagnaIT** [comment]
the website api is not made to works appropriately with android then we are currently limited
what the addon do is a kind of workaround to avoid to implement full android app stack

**Reply by ltai0001** [reply]
The kids menu sends me to the main profile menu. Any chance this can be fixed - really appreciate it

**Reply by CastagnaIT** [reply]
if the website does not restore the disabled switch, nothing can be done
i will try recheck

**Reply by ltai0001** [reply]
I think it is working now
